src/global.scss

```diff
@@ -1,13 +1,37 @@
-@import '@/assets/scss/variables.scss';
-/* IMPORTS */
-@import url('https://fonts.googleapis.com/css?family=Nunito:400,600,700,800&display=swap');
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap');
-@import url('https://fonts.googleapis.com/css?family=Josefin+Sans:300,400,600,700&display=swap');
-@import url('https://fonts.googleapis.com/css?family=Montserrat:400,500,600,700,800&display=swap');
+@import "@/assets/scss/variables.scss";
 
 @media screen and (min-width: 800px) {
-  html, body {
+  html,
+  body {
     font-size: 16px;
     vertical-align: baseline;
   }
-}
\ No newline at end of file
+}
+
+::-webkit-input-placeholder {
+  /* Chrome */
+  color: $secondary;
+}
+:-ms-input-placeholder {
+  /* IE 10+ */
+  color: $secondary;
+}
+::-moz-placeholder {
+  /* Firefox 19+ */
+  color: $secondary;
+  opacity: 1;
+}
+:-moz-placeholder {
+  /* Firefox 4 - 18 */
+  color: $secondary;
+  opacity: 1;
+}
+
+/* OVERWRITES */
```

Comment:
Recommended to have overwrites in another '.scss' file. In the long run, we might need overwrites for different libraries so it would probably make more sense to separate into different files. Could import all into 'global.scss' in the end if easier

Reply:
good idea i'll do that